python_utils.py

```diff
@@ -286,17 +293,20 @@ def url_retrieve(source_url, filename=None):
     Returns:
         urlretrieve. The 'urlretrieve' object.
     """
+    context = ssl.create_default_context(cafile=certifi.where())
     try:
         import urllib
         # Change the User-Agent to prevent servers from blocking requests.
         # See https://support.cloudflare.com/hc/en-us/articles/360029779472-Troubleshooting-Cloudflare-1XXX-errors#error1010. # pylint: disable=line-too-long
         urllib.URLopener.version = (
             'Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:47.0) '
             'Gecko/20100101 Firefox/47.0')
-        return urllib.urlretrieve(source_url, filename=filename)
+        return urllib.urlretrieve(
+            source_url, filename=filename, context=context)
```

Comment:
Adding this `context` here won't be an issue for Linux users, right?

Reply:
I don't think so. Why would it? I haven't tested this on Linux personally, but the CI checks run on Linux and passed

Reply:
I also tried testing it and it works fine for me!